src/Core/Usecase/Notification/UpdateNotification.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Ushahidi Platform Update Notification Use Case
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Usecase\Notification;
+
+use Ushahidi\Core\Data;
+use Ushahidi\Core\Entity;
+use Ushahidi\Core\Usecase\UpdateUsecase;
+
+class UpdateNotification extends UpdateUsecase
```

Comment:
What differs here from the parent UpdateUsecase? - can you call that out in a comment?